### PR TITLE
add xformers 0.0.16 warning message

### PR DESCRIPTION
**Pull request by williamberman**
Add warning message for xformers version 0.0.16 to training scripts

re: https://github.com/huggingface/diffusers/issues/2324

### CODE REVIEW

File: examples/dreambooth/train_dreambooth.py [hunk 1]
```diff
@@ -647,6 +647,13 @@ def load_model_hook(models, input_dir):
 
     if args.enable_xformers_memory_efficient_attention:
         if is_xformers_available():
+            import xformers
+
+            xformers_version = version.parse(xformers.__version__)
+            if xformers_version.micro == 16:
```
Comment (pcuenca) [inline examples/dreambooth/train_dreambooth.py:653]: Maybe check the other components too, so we don't fail in xFormers `0.1.16` ?

**Review by pcuenca — APPROVED** [review]
Thank you!

File: examples/dreambooth/train_dreambooth.py (hunk 1 above)
Comment (patrickvonplaten) [reply examples/dreambooth/train_dreambooth.py:653]: Also let's try to use the same "style" consistently. We always use `package.version.parse(...) < ...` -> let's try to use this everywhere :-)

**Review by patrickvonplaten — APPROVED** [review]
+1 on Pedro's comment, apart from this this LGTM

File: examples/dreambooth/train_dreambooth.py (hunk 1 above)
Comment (williamberman) [reply examples/dreambooth/train_dreambooth.py:653]: Haha yes, that's a good point :grin: !

Comment (williamberman) [reply examples/dreambooth/train_dreambooth.py]: done!

**Pull request merged**